testing/tests/DevExpress.jquery/template.tests.js

```diff
@@ -1,18 +1,24 @@
 "use strict";
 
+SystemJS.config({
+    map: {
+        'jqueryify': SystemJS.map.jquery
```

Comment:
What is this config for?

Reply:
The jquery.tmpl NPM package requires `jqueryify`. But for testing, we need to register the template plugin in our instance of jquery.